CoreScriptsRoot/Modules/PlayerDropDown.lua

```diff
@@ -227,7 +234,7 @@ spawn(function()
 end)
 
 local function isBlocked(userId)
-	if (BlockedList[userId] ~= nil and BlockedList[userId] == true) then
+	if (BlockedList[userId]) then
```

Comment:
what is the content of the value here? basically we are just checking if it isn't nil now, but is that enough?